config/samples/components_v1alpha1_customresourcedefinition.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: components.eks.amazonaws.com/v1alpha1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: foo-crd
+spec:
+  name: foos.blah.blu.com
+  namespace: default
```

Comment:
i think you can remove this namespace in sample file.

Reply:
Done